docs/modules/ROOT/pages/spring-cloud-commons/loadbalancer.adoc

```diff
@@ -232,7 +233,7 @@ public class CustomLoadBalancerConfiguration {
 ----
 
 TIP: For the non-reactive stack, create this supplier with the `withBlockingHealthChecks()`.
-You can also pass your own `WebClient` or `RestTemplate` instance to be used for the checks.
+You can also pass your own `WebClient`, `RestTemplate` or `RestClient` instance to be used for the checks.
```

Comment:
Add a comma after `RestTemplate. The Spring Style Guide calls for serial commas (a comma before the last item in a list).